internal/dvserver/api.go

```diff
@@ -138,25 +138,25 @@ func (s *service) Graph(ctx context.Context, in *Graph_Input) (*Graph_Output, er
 
 	// load tasks
 	if filters.WithFetch && gitHubToken != "" {
-		_, err := dvcore.PullAndSave(filters.Targets, s.h, s.schema, gitHubToken, false, s.opts.Logger)
+		_, err := dvcore.PullAndSave(filters.Targets, s.h, s.schema, gitHubToken, s.opts.TrelloToken, s.opts.TrelloApiKey, false, s.opts.Logger)
 		if err != nil {
 			return nil, fmt.Errorf("pull: %w", err)
 		}
 	}
 
 	var tasks dvmodel.Tasks
-	tasks, err = dvstore.LoadTasks(s.h, s.schema, filters, s.opts.Logger)
+	tasks, err = dvstore.LoadTasks(s.h, s.schema, s.opts.TrelloToken, s.opts.TrelloApiKey, filters, s.opts.Logger)
 	if err != nil {
 		return nil, fmt.Errorf("load tasks: %w", err)
 	}
 
 	// fetch if not already in db
 	if len(tasks) == 0 {
-		_, err := dvcore.PullAndSave(filters.Targets, s.h, s.schema, s.opts.GitHubToken, false, s.opts.Logger)
+		_, err := dvcore.PullAndSave(filters.Targets, s.h, s.schema, s.opts.GitHubToken, s.opts.TrelloToken, s.opts.TrelloApiKey, false, s.opts.Logger)
 		if err != nil {
 			return nil, fmt.Errorf("pull: %w", err)
 		}
-		tasks, err = dvstore.LoadTasks(s.h, s.schema, filters, s.opts.Logger)
+		tasks, err = dvstore.LoadTasks(s.h, s.schema, s.opts.TrelloToken, s.opts.TrelloApiKey, filters, s.opts.Logger)
```

Comment:
Why do you need Token & API KEY for loading from the cayley store ?

Reply:
Yup, this should not happen.

But maybe it's due to a bad design decision I made earlier?

@AbdelkarimBENGRINE can you tell us more?

Reply:
this also surprised me but when I read Trello Api [Doc](https://developer.atlassian.com/cloud/trello/guides/rest-api/authorization/) 
It is said that you have to provide both an api key and token unlike classic Oauth provider. I will look for implementation alternative if it causes too much problem in Depviz